database/migrations/2022_09_20_225438_create_poems_table.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    public function up()
+    {
+        Schema::create('poems', function (Blueprint $table) {
+            $table->id();
+            $table->foreignId('poet_id')
+                ->constrained('poets');
+            $table->integer('translation_of');
```

Comment:
1. translation_of is nullable because "original" poems aren't translations of anything

2. We need to add below a constraint like:
```
Schema::table('poems', function (Blueprint $table) {
           $table->foreign('translation_of')->references('id')->on('poems');
        });
```
        
3. The `id()` method is an alias of the `bigIncrements()`. So, it's UNSIGNED BIGINT. 
=> (see point 2) `translation_of` should be an  UNSIGNED BIGINT as well (`unsignedBigInteger()`)